platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/MerkleDbDataSource.java

```diff
@@ -193,10 +189,10 @@ private static class MetadataFileFormatVersion {
     private final VirtualLeafRecord[] leafRecordCache;
 
     /** ScheduledThreadPool for executing merges */
-    private final ScheduledThreadPoolExecutor mergingExecutor;
+    private final ScheduledThreadPoolExecutor compactingExecutor;
```

Comment:
Eventually, this executor will need to be moved to MerkleDb level (to share across all data sources).

Reply:
Yes, I'll do it in as a part of https://github.com/hashgraph/hedera-services/issues/7502